src/components/SearchResultDetail/SearchResultHtmlModal.tsx

```diff
@@ -0,0 +1,61 @@
+'use client';
+
+import * as React from 'react';
+import Modal from 'react-modal';
+
+import { MODAL_DEFAULT_STYLES } from '@/constants/constants';
+
+interface InjectedProps {
```

Comment:
Why is it caslled `InjectedProps` and not:

```suggestion
interface SearchResultHtmlModalProps {
```